kubernetes/kubernetes_service.go

```diff
@@ -99,11 +98,17 @@ func (in *IstioClient) GetService(namespace, serviceName string) (*v1.Service, e
 	return in.k8s.CoreV1().Services(namespace).Get(serviceName, emptyGetOptions)
 }
 
+// GetPods returns the pods definitions for a given service.
```

Comment:
Misleading comment, as there is not a service as parameter but a labelsSet.

Reply:
done (and squashed in the same time)